svf-llvm/lib/LLVMModule.cpp

```diff
@@ -165,12 +165,16 @@ void LLVMModuleSet::build()
 void LLVMModuleSet::createSVFDataStructure()
 {
     getSVFType(IntegerType::getInt8Ty(getContext()));
-
     for (const Module& mod : modules)
     {
         /// Function
-        for (const Function& func : mod.functions())
+        for (const Function& func2 : mod.functions())
         {
+            const Function* func_ptr = LLVMUtil::getDefFunForMultipleModule(&func2);
+            const Function& func = *func_ptr;
+            if (LLVMFunc2SVFFunc.find(&func) != LLVMFunc2SVFFunc.end()) {
```

Comment:
should be all app's functions' definitions not functions' definitions from ext_api.c